### ISSUE TITLE
BUG: source anchor mangler tekst på noen sider for spesifikasjoner og vokabularer

### ISSUE THREAD

**Issue by KjerstiSteien** [Informasjonsforvaltning/fdk-issue-tracker#887]
## 🐛 Bug rapport
Source anchor mangler tekst på enkelte sider.

### Beskrivelse av feilen fra Monsido:
![image](https://github.com/Informasjonsforvaltning/fdk-issue-tracker/assets/110605570/01027d24-a3ab-4aaa-ba1e-637e752c8228)

### Gjelder følgende sider:
[Veileder for beskrivelse av datasett](https://data.norge.no/guide/veileder-beskrivelse-av-datasett/v1) 
[Forvaltningsstandard for tilgjengeliggjøring av begrepsbeskrivelser basert på TBX (TBX-AP-NO) Versjon 1.0](https://data.norge.no/specification/tbx-ap-no/v1)
[Spesifikasjon for beskrivelse av informasjonsmodeller (ModellDCAT-AP-NO)](https://data.norge.no/specification/modelldcat-ap-nohttps://data.norge.no/specification/modelldcat-ap-no)
[Veileder for beskrivelse av informasjonsmodeller](https://data.norge.no/guide/veileder-modelldcat-ap-no)
[Forvaltningsstandard for begrepsbeskrivelser Versjon 2.0.2](https://data.norge.no/specification/forvaltningsstandard-begrepsbeskrivelser)
[Veileder for beskrivelse av datasett, datatjenester og datakataloger](https://data.norge.no/guide/veileder-beskrivelse-av-datasetthttps://data.norge.no/guide/veileder-beskrivelse-av-datasett)
[Spesifikasjon for beskrivelse av kvalitet på datasett](https://data.norge.no/specification/spesifikasjon-for-beskrivelse-av-kvalitet-pa-datasett)
[Veileder for tilgjengeliggjøring av åpne data](https://data.norge.no/guide/veileder-apne-data)
[Termlosen](https://data.norge.no/specification/termlosen)  
[Forvaltningsstandard for begrepsbeskrivelser (SKOS-AP-NO-Begrep)](https://data.norge.no/specification/skos-ap-no-begrep)
[information-model-type](https://data.norge.no/vocabulary/information-model-type)
[Veileder for beskrivelse av kvalitet på datasett – kvantifiserbar kvalitet](https://data.norge.no/guide/veileder-kvantifiserbar-kvalitet)  
[quality-dimension](https://data.norge.no/vocabulary/quality-dimension)
[quality-metric](https://data.norge.no/vocabulary/quality-metric)
[SKOS-AP-NO-Begrep – Forvaltningsstandard for tilgjengeliggjøring av begrepsbeskrivelser basert på SKOS](https://data.norge.no/specification/skos-ap-no-begrep/v1)
[TBX-AP-NO Forvaltningsstandard for tilgjengeliggjøring av begrepsbeskrivelser basert på TBX](https://data.norge.no/specification/tbx-ap-no)  
[relationship-with-source-type](https://data.norge.no/vocabulary/relationship-with-source-type)  
[audience-type](https://data.norge.no/vocabulary/audience-type)
[Spesifikasjon for klassifikasjonsbeskrivelser (XKOS-AP-NO)](https://data.norge.no/specification/xkos-ap-no)  
[classification-item-change-type](https://data.norge.no/vocabulary/classification-item-change-type)
</html>

### Knyttet til wcag-krav
Level 2.0 (A)
Success criteria 2.4.4

### Hvem avklarer man med?
* Om innhold og utforming av siden: @oystein-asnes og Jim Yang
* Om universell utforming: Spør i kanal #universell-utforming på Altinn devops-slack eller i frontend-forum

**Comment by jeffreiffers** [comment]
Løsningen er å bruke sectlinks istedenfor sectanchors
